Cargo.toml

```diff
@@ -1,16 +1,5 @@
-[package]
-name = "tower-http"
```

Comment:
I think this broke the `tower-grpc` build - that project still contains references to this package.

Reply:
Yep... I messed that up. Fixes are coming.

Reply:
The recent churn is an effort to get the whole thing on crates.io :) so sorry about that.